insights/tests/client/connection/test_test_connection.py

```diff
@@ -449,9 +449,7 @@ def test_legacy_urls_post_all_fails(get, post, exception, insights_connection):
 
 
 @parametrize_methods
-def test_legacy_urls_no_catch(
-    http_method, request_function, insights_connection
-):
+def test_legacy_urls_no_catch(http_method, request_function, insights_connection):
```

Comment:
This is only a result of reformatting by _black_.

Reply:
Please revert this part of the file. Let's not introduce additional changes that aren't related to the main problem. This should be done in a separate PR, if required.

Reply:
Sure thing!

Reply:
In https://github.com/RedHatInsights/insights-core/pull/4211, however, unrelated [pieces](https://github.com/RedHatInsights/insights-core/pull/4211) got merged actually:

* Two kinds of inconsistencies in existing tests.
* A fix of an existing incorrect tests.

I agree though that those should have been and be separate.